02-props-state-component-architecture/props/pokedex/src/pokedex.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import Pokecard from "./Pokecard";
+import "./pokedex.css";
+
+export default class Pokedex extends Component {
+  render() {
```

Comment:
this can also be stateless functional component! But I don't think we taught you that before this exercise was assigned, so it's fine